students/julienjass/Lesson03/Mailroom.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+import sys
+
+# Data base made of a list containing the donors' information
+donor_list = [("William Gates, III", [261514, 392270.49]),
+            ("Mark Zuckerberg", [4918.83, 9837.66, 1639.61]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [354.21, 212.53, 141.68])]
+
+def mainloop():
+    """This function takes no parameters. It contains the main menu prompt which allows the user: with choice 1 to
+    send a Thank You message (calling the function thank_you), with choice 2 to display a report with the donors'
+    information (calling the function report), and with choice 3 to leave the application (calling the function
+    leave)."""
+    print('Mailroom Application')
+    print()
+    response = ''
```

Comment:
No need to initialize this to an empty string here

Reply:
Well noted. Thanks. will fix it and resubmit it with my next PR tonight